src/masternode.h

```diff
@@ -258,6 +259,19 @@ class CMasternode
 
     bool IsWatchdogExpired() { return nActiveState == MASTERNODE_WATCHDOG_EXPIRED; }
 
+    bool IsValidForPayment()
+    {
+        if(nActiveState == MASTERNODE_ENABLED) {
+            return true;
+        }
+        if(!sporkManager.IsSporkActive(SPORK_14_REQUIRE_SENTINEL_FLAG) &&
```

Comment:
Ahh, true, it's me who was blind - too used to "fail early on some conditions and to return true if nothing fails" pattern and completely ignored ENABLED check :D
Still would prefer this function to be written that way but utACK :)